src/components/ChatEntry.js

```diff
@@ -1,22 +1,35 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp.js';
+
 
 const ChatEntry = (props) => {
+  const localSender = props.sender === 'Vladimir';
+
+  const handleClick = () => {
+    props.onLike();
+  }
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={`chat-entry ${localSender ? 'local' : 'remote'}`}>
+      <h2 className="entry-name">{props.sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{props.body}</p>
+        <p className="entry-time"><TimeStamp time = {props.timeStamp}/></p>
+        <button className="like" onClick={handleClick}>
```

Comment:
```suggestion
        <button className="like" onClick={props.onLike}>
```
You could have the on click event call `props.onLike()` directly here. That way, you don't need to set up another function for `handleClick()`